app/assets/stylesheets/views/_topical_events.scss

```diff
@@ -9,6 +9,7 @@
   .organisations-list {
     list-style: none;
     /* stylelint-disable */
+    padding-inline-start: 0px;
```

Comment:
I don't think this should be within the stylelint-disable block

This is also newer CSS that IE doesn't support so feels a bit funky. I wonder if there's a more basic CSS approach to solve the problem.

Also for quite a nuanced fix there isn't much information in the commit message.

Reply:
Agree that it would be good to know more about the issue we're trying to solve, and also to get a URL for a page that this affects.

Reply:
Thanks both, this now updated to just be padding: 0 and be outside the stylelint disable block. I've also beefed up the commit description, so considering this set of comments resolved 🙂